### PR TITLE
Add UI for pausing on failure via developer mode

### PR DESCRIPTION
**Pull request by Martchus**
* Add checkbox to enable/disable whether to pause on failures
* Add button to resume ignoring the failure
* Related ticket: https://progress.opensuse.org/issues/119380

### CODE REVIEW

**Comment by okurz** [comment]
screenshot pretty plz :)

**Review by okurz — APPROVED** [review]

**Comment by Martchus** [comment]
screenshot_20221122_124215

**Comment by Martchus** [comment]
Keeping it as a draft until the os-autoinst PR has been merged. It wouldn't cause much trouble merging this PR independently. The new UI elements would simply have no effect. However, I suppose it is better to avoid confusing users.

The hadolint problem that I've consistently noticed after pushing multiple times is clearly unrelated. I've created a separate PR to fix it (https://github.com/os-autoinst/openQA/pull/4914).

**Comment by kalikiana** [comment]
@mergifyio rebase

File: t/ui/25-developer_mode.t [hunk 1]
```diff
@@ -491,30 +494,43 @@ subtest 'start developer session' => sub {
...
+        $script .= '{ data: "{\"type\":\"info\",\"what\":\"cmdsrvmsg\",\"data\":{\"';
+        $script .= $response_message;
+        $script .= '\":0}}" });';
+        $driver->execute_script($script);
+        is js_variable("developerMode.$deveoper_mode_property"), 0, 'property unset again';
+        ok !element_prop($checkbox_id, 'checked'), 'checkbox is unchecked again';
+    };
+    subtest 'select whether to pause on next command' => sub {
+        $test_boolean_behavior_switch->(
+            pauseOnNextCommand => 'set_pause_on_next_command',
```
Comment (kalikiana) [inline t/ui/25-developer_mode.t:525]: I find it odd to use `=>` here but I guess it's just a `,` afterall.

**Review by kalikiana — APPROVED** [review]

**Pull request merged**